openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -94,9 +94,10 @@ def get_all_orgs(cls):
         org_filter_set = set()
 
         for configuration in cls.objects.filter(values__contains='course_org_filter', enabled=True).all():
-            org_filter = configuration.get_value('course_org_filter', None)
-            if org_filter:
-                org_filter_set.add(org_filter)
+            org_filter = configuration.get_value('course_org_filter', [])
```

Comment:
No, we are not dealing with the current site here. We are checking the value on a given SiteConfiguration model.